fssb.c

```diff
@@ -67,23 +67,23 @@ int handle_syscalls(pid_t child) {
 
     int syscall = get_reg(child, orig_eax);
 
-    if(syscall != SC_EXECVE && first_rxp_mem == -1) {
+    if(syscall != SYS_execve && first_rxp_mem == -1) {
         first_rxp_mem = get_readonly_mem(child);
         int i;
         for(i = 0; i < 6; i++)
             write_slots[i] = first_rxp_mem + i*RDONLY_MEM_WRITE_SIZE;
     }
 
     switch (syscall) {
-        case SC_EXIT:
-        case SC_EXIT_GROUP: {
+        case SYS_exit:
+		case SYS_exit_group: {
```

Comment:
I think there's a change from spaces to tab for indentation over here. Probably missed your eye :)